tests/tests_app/storage/test_payload.py

```diff
@@ -142,6 +142,7 @@ def run(self):
             self.stop()
 
 
+@pytest.mark.skipif(True, reason="flaky")
```

Comment:
```suggestion
@pytest.mark.xfail(strict=False, reason="flaky")
```
or delete the test completely

Reply:
@Borda Those features from Lightning App required for the platform and they are flaky. It is fine to skip them for now. We will repair them if this becomes a priority in the future.

Reply:
that is fine to drop them but do it transparent way not as skipif-true